### DISCUSSION TITLE
How to use naga in other system like OpenHarmony Operating System（ohos）in mobilephone?

### DISCUSSION THREAD

**Discussion by lili-0106** [gfx-rs/wgpu#6117] [Q&A] [answered]
Is naga designed and written for windows? can we use naga independently in ohos(OpenHarmony Operating System) ? if can, can i have a reference? how to compile naga in that system?

**Comment by cwfitzgerald** [comment, marked as answer]
Naga is system-independent rust, so it will run anywhere rust can run. As for how to build for that platform, we can't help there, but it seems to be possible.

**Reply by ErichDonGubler** [reply]
In addition to @cwfitzgerald's answer, it looks like there might be a lead for OpenHarmony support in upstream Rust: https://doc.rust-lang.org/rustc/platform-support/openharmony.html
